code/game/objects/items/devices/flashlight.dm

```diff
@@ -147,6 +147,16 @@
 	w_class = 1
 	raillight_compatible = FALSE
 
+/obj/item/device/flashlight/combat //as bright as rail lights, don't see why you need them to be mounted on a gun to be brighter
+	name = "combat flashlight"
+	desc = "A hand-held flashlight, with increased light for military usage."
+	icon = 'icons/obj/lighting.dmi'
+	icon_state = "flashlight_combat"
```

Comment:
The shape is a little off, it should look just a little more like the seclite's sprite of other stations, especially around the lens.

Reply:
Should I just port the seclite then? I'm not a good spriter.

Reply:
Feel free too, honestly these flashlight sprites are kinda untasteful compared to the vanilla spaceman ones